server/src/main/java/org/elasticsearch/health/RestGetHealthAction.java

```diff
@@ -33,7 +33,7 @@ public String getName() {
 
     @Override
     public List<Route> routes() {
-        return List.of(new Route(GET, "/_internal/_health"), new Route(GET, "/_internal/_health/{indicator}"));
+        return List.of(new Route(GET, "/_health"), new Route(GET, "/_health/{indicator}"));
```

Comment:
Out of curiosity, should we just *add* `/_health` and `/_health/{indicator}` to the paths so we have a period during which both URLs work? Then at a later time (9.x?) we can remove the `/_internal/health` path?

Reply:
That's a very interesting idea.
I do worry that not ALL our `_internal` APIs will be internal/experimental/not-GA anymore should we choose to do this though. The current invariant of `DO NOT USE` for the `_internal` APIs (irrespective of reading the docs) is quite valuable IMO.